api/inventorypb/agent_status.proto

```diff
@@ -7,16 +7,16 @@ option go_package = "api/inventorypb;inventorypb";
 // AgentStatus represents actual Agent status.
 enum AgentStatus {
   AGENT_STATUS_INVALID = 0;
-  // Agent is starting.
-  STARTING = 1;
   // Agent is running.
-  RUNNING = 2;
+  RUNNING = 1;
+  // Agent is starting.
+  STARTING = -1;
   // Agent encountered error and will be restarted automatically soon.
-  WAITING = 3;
+  WAITING = -2;
   // Agent is stopping.
-  STOPPING = 4;
+  STOPPING = -3;
   // Agent finished.
-  DONE = 5;
+  DONE = -4;
   // Agent is not connected, we don't know anything about it's state.
-  UNKNOWN = 6;
+  UNKNOWN = -5;
```

Comment:
it's a breaking change. please revert it.

Reply:
done